transformers_sklearn/transformers_sklearn.ipynb

```diff
@@ -0,0 +1,754 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "d8bd96c9",
+   "metadata": {},
+   "source": [
+    "# Training Transformer Models with Scikit-learn"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a60a726b",
+   "metadata": {},
+   "source": [
+    "[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/cleanlab/examples/blob/master/transformers_sklearn/transformers_sklearn.ipynb)     "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3b13d8a2",
+   "metadata": {},
+   "source": [
+    "Have you ever wanted to use some handy scikit-learn functionalities with complex neural network models but could not because TensorFlow models are not compatible with the scikit-learn API? Here we introduce Cleanlab's KerasWrapperModel, a scikit-learn compatible wrapper for TensorFlow/Keras models that allows the usage of TensorFlow's neural network models with scikit-learn features such as Pipeline, GridSearch and more.\n",
+    "\n",
+    "To demonstrate how we can use this KerasWrapperModel, we will train a classifier (using a pretrained bert model) to classify positive and negative reviews on an Amazon Reviews dataset following the steps below:\n",
+    "\n",
+    "- Tokenize out text data using a tokenizer from a pretrained bert model \n",
+    "- Define our neural network model and wrap it using cleanlab's `KerasWrapperModel`\n",
+    "- Conduct a grid search to find the best parameters for the model using sklearn's `GridSearchCV`\n",
+    "- Using the optimal parameters found, train the classifier to classify the reviews\n",
+    "- To further improve the model performace, use cleanlab's `CleanLearning` to identify potential label issues in the dataset and retrain the classifier to get improved predictions"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "56d44364",
+   "metadata": {},
+   "source": [
+    "Please install the dependencies specified in this [requirements.txt](https://github.com/cleanlab/examples/blob/master/transformers_sklearn/requirements.txt) file before running the notebook.  "
```

Comment:
also clarify this notebook should ideally be run with GPU for better efficiency?